source/includes/steps-fle-convert-to-a-remote-master-key-gcp.yaml

```diff
@@ -132,6 +125,28 @@ content: |
                    "endpoint": "<GCP authentication endpoint>",
                }
            }
+
+     .. tab::
+        :tabid: csharp
+
+        .. code-block:: csharp
+
+           var kmsProviders = new Dictionary<string, IReadOnlyDictionary<string, object>>();
+
+           var gcpPrivateKey = Environment.GetEnvironmentVariable("FLE_GCP_PRIVATE_KEY");
+           var gcpEmail = Environment.GetEnvironmentVariable("FLE_GCP_EMAIL");
+           var gcpEndpoint = Environment.GetEnvironmentVariable("FLE_GCP_IDENTITY_ENDPOINT"); // Optional, defaults to "oauth2.googleapis.com".
+           var gcpKmsOptions = new Dictionary<string, object>
+           {
+               { "privateKey", gcpPrivateKey },
+               { "email", gcpEmail },
+           };
+           if (gcpEndpoint != null)
```

Comment:
Does it still work if, for one of the optional parameters, you have a key, but no value? It could be easier to read without this conditional. Another idea: include it in a commented line as most users shouldn't need to update the default value.

Reply:
> you have a key, but no value?

if I understood the question correctly, it's not what happens here. If the value is not presented, then the key won't be added

Reply:
up to you guys